lib/services.js

```diff
@@ -200,6 +201,7 @@ const services = {
             md.setAmzRestore({
                 'ongoing-request': false,
                 'expiry-date': archive.restoreWillExpireAt,
+                'content-md5': restoredEtag,
```

Comment:
will this field be "discarded" automatically when `restoredEtag` is not provided (e.g. when it matches)?
or should we always set `restoredEtag` ?

Reply:
It gets discarded from the metadata when not provided.